setup.py

```diff
@@ -55,18 +55,18 @@
         'rules',
         'zipstream==1.1.4',
         # Production-only
-        'django-composed-configuration[prod]>=0.19.2',
+        'django-composed-configuration[prod]==0.21.0',
```

Comment:
Alternatively, we could pin this to the version before Django 4 support was required, and that would prevent us from having to pin Django to >=4 and also let us leave the RGD dependencies as-is.

Reply:
I'd say go ahead and go all-in on Django4

Reply:
I don't see a reason to pin this but rather a `>=0.21.0` should be fine, right?

Reply:
Like you have in the `dev` extras